src/plugins/data/public/autocomplete/providers/value_suggestion_provider.ts

```diff
@@ -85,6 +96,16 @@ export const setupValueSuggestionProvider = (
       : undefined;
     const filterQuery = timeFilter ? buildQueryFromFilters([timeFilter], indexPattern).filter : [];
     const filters = [...(boolFilter ? boolFilter : []), ...filterQuery];
-    return await requestSuggestions(title, field, query, filters, signal);
+    try {
+      usageCollector?.trackCall();
+      return await requestSuggestions(title, field, query, filters, signal);
+    } catch (e) {
+      if (!signal?.aborted) {
+        usageCollector?.trackError();
```

Comment:
nit: I think this also belongs to `requestSuggestions` and should be similar to `usageCollector?.trackResult();` call

Reply:
I agree, but I didn't want to delete anything from `requestSugestions.cache`  before the function returned.